app.js

```diff
@@ -1,10 +1,14 @@
-// require necessary modules
 var express = require('express')
-  , mongoskin = require('mongoskin')
-  , routes = require('./routes')
-  , http = require('http')
-  , path = require('path')
-  , db = mongoskin.db((process.env.MONGOLAB_URI || 'localhost:27017/test'), {safe:true});
+  , rtIndex  = require('./routes')
```

Comment:
srry 4 abbr. Started programming in VB w/ Hungarian Notation. nounEverything verbGets adjPrefixed. Thx.